lib/setup/setupInstall.js

```diff
@@ -1131,12 +1136,24 @@ function Install(options) {
                                 obj.ts = Date.now();
                                 objects.setObject(obj._id, obj, (err /*, res */) => {
                                     if (err) {
-                                        console.error('host.' + hostname + ' error: ' + err);
+                                        console.error(`host.${hostname} error: ${err}`);
                                     } else {
-                                        console.log('host.' + hostname + ' object ' + obj._id + ' created');
+                                        console.log(`host.${hostname} object ${obj._id} created`);
                                     }
                                     setTimeout(setObjs, 25);
                                 });
+                            } else {
+                                setStates();
+                            }
+                        }
+
+                        // sets the default states if any given
+                        function setStates() {
+                            if (defStates.length > 0) {
+                                const defState = defStates.pop();
+                                states.setState(defState.id, defState.val);
```

Comment:
setTimeout should be done in callback from setState :-)

Reply:
... also the logging

Reply:
true, wrote promise-style first, then switched to cbs..